### ISSUE TITLE
Path Support and image base on Area

### ISSUE THREAD

**Issue by anhnvme** [junalmeida/homeassistant-minimalistic-area-card#3]
Hello,

Can you add path support like base card. ?  I want to link to a new URL 
Secondly, can you use image from area without insert images url ?

thank for create awesome card.

**Comment by junalmeida** [comment]
Not sure I got right your request, could you provide some example? (maybe a proposal to the yaml)? 

I'm not sure you can use the image from the new added area feature, I can certainly take a look on that. Please open up a separate ticket for each question. 

Thx

**Comment by tm24fan8** [comment]
For the path support, I believe they're talking about how the stock HA area card allows you to link to a different page in your Lovelace UI. 

For example, I click the "basement" card and it sends me to the Lovelace page containing more options for my basement. That sort of thing.

**Comment by tm24fan8** [comment]
Ah, I see someone else already mentioned this in #2 and it looks like you're already planning to add it. Nice :)

**Comment by junalmeida** [comment]
Yes, if you mean the `tap_action`, you may close this one as it will be tracked on #2. Thx

**Comment by anhnvme** [comment]
@junalmeida yes, it's that mean link to other page. 
Thank you

**Comment by anhnvme** [comment]
About area images, it's that mean, like a area card at home assistant, it's will use image already uploaded from area at home assistant
![image](https://user-images.githubusercontent.com/13740754/167524025-981dca94-73a2-48b8-aee4-807e831b7e66.png)
